src/Administration/Resources/app/administration/src/module/sw-plugin/view/sw-plugin-list/index.js

```diff
@@ -378,13 +385,13 @@ Component.register('sw-plugin-list', {
         isConfigAvailableForPlugins() {
             this.isLoading = true;
 
-            return Promise.all(this.plugins.map((plugin) => {
+            this.plugins.map((plugin) => {
```

Comment:
You create a new variable and don´t return it. Whats the purpose of this? :) And also this could leak to potential breaks because the methods return previously a Promise and now is a void function